src/Plugin_Command.php

```diff
@@ -498,6 +498,9 @@ protected function install_from_repo( $slug, $assoc_args ) {
 	 *
```

Comment:
Can you include functional tests for using this argument with both themes and plugins?